lib/_stream_transform.js

```diff
@@ -92,7 +96,7 @@ function afterTransform(stream, er, data) {
     return stream.emit('error', new Error('no writecb in Transform class'));
 
   ts.writechunk = null;
-  ts.writecb = null;
+  ts.writecb = emitMultipleCallsError.bind(stream);
```

Comment:
Note. If this PR gets accepted, this `bind()` could be moved to, for example, the stream constructor, and hidden with a symbol. That would avoid rebinding every time.